Core/clim-basic/decls.lisp

```diff
@@ -231,6 +231,20 @@ NEW-NAME must be a string.
 Changing the pretty name of SHEET usually changes the title of the
 window associated with it."))
 
+(defgeneric clime:sheet-icon (sheet)
```

Comment:
please remove package accessors - which symbol belongs to which package is decided in the file packages.lisp (and climi imports exported packages).

Reply:
Changed.